build/webpack/vendor.js

```diff
@@ -0,0 +1,10 @@
+//Add to this file any modules that are used across many of your component bundles.
+
+const React = require("react"),
+    ReactDom = require("react-dom"),
+    FinsembleControls = require("@chartiq/finsemble-react-controls"),
```

Comment:
yeah. But each of the sample components uses the react-controls, so they would break too. If the client didn't have the controls, they could just delete this line.